lua/wikis/commons/Infobox/Person.lua

```diff
@@ -34,8 +34,16 @@ local Center = Widgets.Center
 local Builder = Widgets.Builder
 local Customizable = Widgets.Customizable
 
+---@class PersonRoleData
+---@field category string?
+---@field display string?
+
 ---@class Person: BasicInfobox
 ---@field locations string[]
+---@field role PersonRoleData? #deprecated
+---@field role2 PersonRoleData? #deprecated
+---@field role3 PersonRoleData? #deprecated
```

Comment:
If i am not mistaken all usage is going to be removed with this PR, so might as well drop them.